Cards/CardBase.gd

```diff
@@ -35,12 +35,41 @@ func _apply_all_effects(target: Entity) -> void:
 	for effect_data: EffectData in card_effects_data:
 		effect_data.apply_effect_data(target)
 
+func _apply_effect(target : Entity, effect: EffectData):
```

Comment:
`EffectBase` already has an `apply_effect` function, and this is not doing much, so I don't really see the goal of this function

Reply:
That's fair, I'll change it